### ISSUE TITLE
Conversion SR parameters to other software

### ISSUE THREAD

**Issue by Andrej730** [DummkopfOfHachtenduden/SilkroadDoc#12]
1. Any information about how SR parameters (such as normals, position, uv) relate to other software? 

2. Is there any other factors besides normal value in BMS format that affect how normals will work in game? 
I made custom bms file for tests (attached it here - [chinaman_adventurer_face.bms.txt](https://github.com/DummkopfOfHachtenduden/SilkroadDoc/files/10096370/chinaman_adventurer_face.bms.txt) ), 

I've used the guidelines described below but it still looks broken in game. I've tried to inverse axes during normal calculation but got the same results. 

In-game:
![image](https://user-images.githubusercontent.com/9417531/204091875-4d077096-04ba-4f56-ae7c-5c4a963fc3dd.png)

Blender:
![image](https://user-images.githubusercontent.com/9417531/204091893-8e7684ba-aab0-4bd0-8ee2-a0c08f2269c9.png)

What I've learned experimentingis that SR has left-handed coordinate system with Y up. 
So the formulas to convert parameters to Blender (right-handed coordinate system with Z up) are:
1) Blender Position = SR Postion.xzy * Vector(-1, 1,  -1)
2) Blender Normal = SR Normal.xzy * Vector(-1, 1, -1)
3) Blender UV = (SR UV.x, 1 - SR UV.y)

Maybe we should put it somewhere on the [wiki](https://github.com/DummkopfOfHachtenduden/SilkroadDoc/wiki)?

**Comment by Andrej730** [comment]
Update 1: I've noticed that winding order in SR is left-handed (clockwise) meaning it expects for all visible faces winding order to be like on the second triangle. And the reason why I have this problem with normals - some of the faces have counter-clockwise order (first triangle). 

Looking for a way how to detect that and make winding order for all faces to be clockwise.

![image](https://user-images.githubusercontent.com/9417531/204101060-788b6435-40ec-4ff6-95c0-841fbfc1e720.png)

Update 2: Found the solution - my modeling program (Blender) is using counter-clockwise winding order for faces and it's keeping it automatically. So the solution is to just reverse it exporting winding order to SR.
